yaml_decode/example/example.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:json_annotation/json_annotation.dart';
```

Comment:
This example doesn't directly use any apis from this package?

Reply:
The annotation, my friend. 😉

Reply:
I get that JsonSerializable uses it, but that doesn't serve as an example for using this package?

Reply:
Sorry. Doing crawl walk run. Going to add in a CLI here that parses yaml.

Reply:
...and associated tests, etc

Reply:
Is it worth having this at all for now though?

Reply:
Trying to keep the PRs smaller. I'm going to build on this next – I promise. 😄